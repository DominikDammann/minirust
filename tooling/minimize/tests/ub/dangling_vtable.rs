fn foo(_x: *const dyn std::fmt::Debug) {}

fn main() {
    // cannot use `std::mem::zeroed`: unsupported Rust intrinsic `write_bytes`.
    foo(unsafe { core::mem::transmute::<[usize; 2], *const dyn std::fmt::Debug>([0_usize; 2]) });
}
