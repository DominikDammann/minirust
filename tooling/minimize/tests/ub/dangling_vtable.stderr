fatal error: UB: invalid pointer for vtable lookup
